packages/remix-dev/modules.ts

```diff
@@ -68,6 +68,10 @@ declare module "*.ttf" {
   let asset: string;
   export default asset;
 }
+declare module "*.wasm" {
```

Comment:
Will these need to be copied to the server output directory for node or will they be loaded from the node_modules folder?